nowcasting_dataloader/datamodules.py

```diff
@@ -0,0 +1,156 @@
+"""Datamodules to use for training"""
+import logging
+import os
+from typing import List, Optional, Tuple, Union
+
+import torch
+from nowcasting_dataset.config import load_yaml_configuration
+from nowcasting_dataset.config.model import Configuration
+from pytorch_lightning import LightningDataModule
+
+from nowcasting_dataloader.datasets import SatFlowDataset, worker_init_fn
+
+_LOG = logging.getLogger(__name__)
+_LOG.setLevel(logging.DEBUG)
```

Comment:
might want to remove this, let hte user decide what they want

Reply:
Okay, sounds good!